src/relion/zocalo/wrapper.py

```diff
@@ -328,6 +333,28 @@ def get_status(self, job_path):
             # if job_finished_file exists:
 
 
+@functools.singedispatch
+def images_msgs(relion_stage_object, job_string: str):
+    logger.debug(f"{relion_stage_object!r} does not have associated images")
+    return []
+
+
+@images_msgs.register(relion.MotionCorr)
+def _(stage_object: relion.MotionCorr, job_string: str):
+    imgs = []
+    for micrograph in stage_object[job_string]:
+        imgs.append(micrograph.micrograph_snapshot_full_path.replace(".jpeg", ".mrc"))
+    return imgs
```

Comment:
Recommend using list comprehensions where possible over appending elements in a loop
```suggestion
    return [
        micrograph.micrograph_snapshot_full_path.replace(".jpeg", ".mrc")
        for micrograph in stage_object[job_string]
    ]
```